src/main/java/com/google/api/codegen/py/PythonGapicContext.java

```diff
@@ -250,23 +256,30 @@ private String returnTypeComment(
 
     // parameter types
     contentBuilder.append("Args:\n");
-    for (Field field :
-        removePageTokenFromFields(method.getInputType().getMessageType().getFields(), config)) {
-      String name = pythonCommon.wrapIfKeywordOrBuiltIn(field.getSimpleName());
-      if (config.isPageStreaming()
-          && field.equals((config.getPageStreaming().getPageSizeField()))) {
-        contentBuilder.append(
-            fieldComment(
-                name,
-                field,
-                importHandler,
-                "The maximum number of resources contained in the\n"
-                    + "underlying API response. If page streaming is performed per-\n"
-                    + "resource, this parameter does not affect the return value. If page\n"
-                    + "streaming is performed per-page, this determines the maximum number\n"
-                    + "of resources in a page."));
-      } else {
-        contentBuilder.append(fieldComment(name, field, importHandler, null));
+    if (method.getRequestStreaming()) {
+      contentBuilder.append(
+          "  requests (iterator of "
```

Comment:
"iterator of xxx" --> "Iterator[xxx]"

(https://www.python.org/dev/peps/pep-0484/)